teacher_t.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(name, age, parent_permission, specialization)
```

Comment:
- Please `name` and `parent_permission` are optional and have default values of "Unknown" and "true". [Refer project requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md)